app/components/retriable_fast_image/index.tsx

```diff
@@ -30,12 +44,20 @@ export default class RetriableFastImage extends PureComponent<RetriableFastImage
     }
 
     render() {
-        return (
-            <FastImage
-                {...this.props}
-                key={`${this.props.id}-${this.state.retry}`}
-                onError={this.onError}
-            />
-        );
+        if (this.state.valid) {
+            return (
+                <FastImage
+                    testID={`${this.props.id}-${this.state.retry}`}
+                    onError={this.onError}
+                    {...this.props}
+                />
+            );
+        }
+
+        if (this.props.renderOnError) {
+            return (<Image source={{}}/>);
```

Comment:
I believe the broken image is from compass Icon. Maybe search the codebase for `jumbo-attachment-image-broken` or `image-broken-outline`